src/Bicep.Core/Extensions/DateTimeExtensions.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Xml;
+
+public static class DateTimeExtensions
+{
+    public static string ToRFC3339(this DateTime date)
```

Comment:
nit: According to Only [Capitalization Rules for Identifiers](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers), we should only capitalize two-letter acronyms.
```suggestion
    public static string ToRfc3339Format(this DateTime date)
```